### PR TITLE
add seconds and minutes, as well as sound fx on finish

### PR DESCRIPTION
**Pull request by luketurnerdev**

### CODE REVIEW

File: focus.sh [hunk 1]
```diff
@@ -11,12 +10,15 @@ sudo ifconfig en0 down
 sudo ifconfig en0 up
 
 SECONDS_LEFT=$(( ${1:-25} * 60 ))
+HRS_LEFT=$(( ${1:-25} / 60 ))
+MINS_LEFT=$(( ${1:-25} ))
```
Comment (onesc) [inline focus.sh:16]: mins and hrs wont recalculate I think you need to do it in the loop

File: focus.sh [hunk 2]
```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-# append blocked websites to /etc/hosts
```
Comment (onesc) [inline focus.sh:3]: can we keep this comment?

**Review by onesc — CHANGES_REQUESTED** [review]

File: focus.sh (hunk 2 above)
Comment (luketurnerdev) [reply focus.sh:3]: no fuck u. this repo is mine now bitch

File: focus.sh (hunk 1 above)
Comment (luketurnerdev) [reply focus.sh:16]: fixed

File: focus.sh (hunk 2 above)
Comment (luketurnerdev) [reply focus.sh:3]: jks lov u

**Review by onesc — APPROVED** [review]

**Pull request merged**